### PR TITLE
Release notes + fixing up nextcloud tutorial

### PR DESCRIPTION
**Pull request by danielobyrne**

### CODE REVIEW

File: source/tutorials/other/deploying-nextcloud.rst [hunk 1]
```diff
@@ -2,19 +2,17 @@
...
 
+This tutorial assumes that you have the following prepared:
 
+* An SSH key.
 
-This tutorial assumes the following:
+* A domain name.
 
-* Have an SSH key.
+* A record pointing to localhost (127.0.0.1) under a chosen host name.
```
Comment (flashvoid) [inline source/tutorials/other/deploying-nextcloud.rst:11]: Record is not necessary, ability to configure domain name entries programmatically. (e.g. create DDNS entry if using namecheap.com for dns)

File: source/tutorials/other/deploying-nextcloud.rst [hunk 2]
```diff
@@ -2,19 +2,17 @@
...
 
-* Have a domain name.
+* Knowledge of how to create and use volumes (if you are using your own volume).
 
-* An A record pointing to localhost (127.0.0.1) under a chosen host name.
-
-* Completed creating and using volumes (if using own volume).
-
-* Have a volume created for storing Nextcloud data (if using terraform).
+* A volume created for storing Nextcloud data (if you are using terraform).
```
Comment (flashvoid) [inline source/tutorials/other/deploying-nextcloud.rst:15]: (Optional)

File: source/tutorials/other/deploying-nextcloud.rst [hunk 3]
```diff
@@ -2,19 +2,17 @@
...
+* An SSH key.
 
-This tutorial assumes the following:
+* A domain name.
 
-* Have an SSH key.
+* A record pointing to localhost (127.0.0.1) under a chosen host name.
 
-* Have a domain name.
+* Knowledge of how to create and use volumes (if you are using your own volume).
```
Comment (flashvoid) [inline source/tutorials/other/deploying-nextcloud.rst:13]: (Optional)

**Review by flashvoid — CHANGES_REQUESTED** [review]

File: source/tutorials/other/deploying-nextcloud.rst (hunk 3 above)
Comment (danielobyrne) [reply source/tutorials/other/deploying-nextcloud.rst:13]: I've added a note about these being optional.

**Review by flashvoid — APPROVED** [review]

**Pull request merged**